prometheus_test.lua

```diff
@@ -189,6 +190,75 @@ function TestPrometheus:testCounters()
   luaunit.assertEquals(self.dict:get("metric1"), 5)
   luaunit.assertEquals(self.dict:get('metric2{f2="v2",f1="v1"}'), 4)
   luaunit.assertEquals(ngx.logs, nil)
+end
+function TestPrometheus:testGaugeIncDec()
+  self.gauge1:inc(-1)
```

Comment:
I think it's OK to allow gauges to be incremented/decremented by negative values.